apps/analysis_framework/filter_set.py

```diff
@@ -86,3 +87,16 @@ def filter_recently_used(self, queryset, name, value):
             entries_qs = Entry.objects.filter(modified_at__gte=recent_usage_cutoff)
             return queryset.filter(id__in=entries_qs.values('analysis_framework'))
         return queryset
+
+
+class AnalysisFrameworkRoleGqFilterSet(UserResourceGqlFilterSet):
+    search = django_filters.CharFilter(method='search_filter')
+
+    class Meta:
+        model = AnalysisFrameworkRole
+        fields = ('__all__')
+
+    def search_filter(self, qs, _, value):
+        if value:
+            return qs.filter(title__icontains=value)
+        return qs
```

Comment:
Not used or required for a bound data.

Reply:
`AnalysisFrameworkRoleGqFilterSet` is not used 
i have to remove it